src/commands/info.rs

```diff
@@ -25,21 +32,28 @@ impl InfoCommand {
 
         println!("registry: {url}", url = client.url());
         println!("\npackages in client storage:");
```

Comment:
Isn't this header incorrect with --namespaces? Maybe the whole thing below should be inverted:

```suggestion
        if self.namespaces {
            Self::print_namespace_map(&client).await?;
            return Ok(());
        }
        println!("\npackages in client storage:");
```